tests/integration/ha_tests/helpers.py

```diff
@@ -38,7 +57,10 @@ async def get_application_name(ops_test: OpsTest, application_name: str) -> str:
     for application in ops_test.model.applications:
         # note that format of the charm field is not exactly "mongodb" but instead takes the form
         # of `local:focal/mongodb-6`
-        if application_name in status["applications"][application]["charm"]:
+        if (
+            application_name in status["applications"][application]["charm"]
+            and application not in EXCLUDED_APPS
+        ):
```

Comment:
`anothr-database-a` would usually be returned first and that would mess with other tests.